### ISSUE TITLE
Improve log in and log out method 

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/repo#540]
_From [mail.Haoqiang@gmail.com](https://code.google.com/u/mail.Haoqiang@gmail.com/) on January 24, 2013 11:40:40_

What can be improved on the system? What can be added to the system? Currently we use app engine built in methods to keep track of users state. But when user logout, the userService.createLogoutURL(redirectPage); will log out the user's google account rather than log out only this system. Please specify: If the user is logged in by app engine, then we start a session. 
When the user click on log out, we just destroy the session instead of log out google.
In conclusion, users should be the same state before login and after logout! We should create a session for the current user.

_Original issue: http://code.google.com/p/teammatespes/issues/detail?id=541_
